test/filter_builder_test.dart

```diff
@@ -5,10 +5,11 @@ void main() {
   group('Filter builder', () {
     group("Attributes", () {
       test('basic transform', () {
-        expect(Meili.attr('book_id').transform(), equals("book_id"));
-        expect(Meili.attr('book.id').transform(), equals("book.id"));
-        expect(Meili.attr('   book.  id   ').transform(), equals("book.id"));
-        expect(Meili.attr('   book.id.   ').transform(), equals("book.id"));
+        expect(Meili.attr('book_id').transform(), equals("\"book_id\""));
+        expect(Meili.attr('book.id').transform(), equals("\"book.id\""));
+        expect(
+            Meili.attr('   book.  id   ').transform(), equals("\"book.id\""));

```

Comment:
```suggestion
        expect(Meili.attr(' book. id ').transform(), equals("\"book.id\""));
```
this way it will fit in the same line 😋

Reply:
trailing commas for the win